tests/test_dulwich.py

```diff
@@ -8,12 +8,18 @@
 import asyncssh
 import paramiko
 import pytest
-from dulwich.contrib.test_paramiko_vendor import CLIENT_KEY, PASSWORD, USER, Server
 from pytest_mock import MockerFixture
 from pytest_test_utils.waiters import wait_until
 
 from scmrepo.git.backend.dulwich.asyncssh_vendor import AsyncSSHVendor
 
+from .vendor.test_paramiko_vendor import (
```

Comment:
C: temporary solution, since something has changed in a way dulwich is packaging its tests, it's not critical, and we need to release it asap I think. I'll look later into migrating back to dulwich provided utils.